api/envoy/config/core/v3/proxy_protocol.proto

```diff
@@ -47,4 +56,10 @@ message ProxyProtocolConfig {
   // This config controls which TLVs can be passed to upstream if it is Proxy Protocol
   // V2 header. If there is no setting for this field, no TLVs will be passed through.
   ProxyProtocolPassThroughTLVs pass_through_tlvs = 2;
+
+  // Custom TLV metadata to be sent in the upstream PROXY protocol header. This field
+  // is only valid when the version is V2. These TLVs can be defined both in the
```

Comment:
Please document (and test) the precedence behavior for a TLV defined in both pass_through and this new field.

Reply:
Took a stab at fleshing out the proto definition to address both your review comments surrounding this area in https://github.com/envoyproxy/envoy/pull/37591/commits/e50c959cf46fc35e54faaff24963f931944ddfb4. Let me know if you think that it needs any further clarification.